source/_integrations/youtube.markdown

```diff
@@ -0,0 +1,68 @@
+2. The wizard will ask you to choose a project to manage your application. Select a project and click continue.
+3. Verify that your YouTube Data API v3 was enabled and click 'Go to credentials'
+4. Navigate to APIs & Services (left sidebar) > [Credentials](https://console.cloud.google.com/apis/credentials)
+5. Click on the field on the left of the screen, **OAuth Consent Screen**.
+6. Select **External** and **Create**.
+7. Set the *App Name* (the name of the application asking for consent) to anything you want, e.g., *Home Assistant*.
+8. You then need to select a *Support email*. To do this, click the drop-down box and select your email address.
+9. You finally need to complete the section: *Developer contact information*. To do this, enter your email address (the same as above is fine).
+10. Scroll to the bottom and click **Save and Continue**. You don't have to fill out anything else, or it may enable additional review.
+11. You will then be automatically taken to the Scopes page. You do not need to add any scopes here, so click Save and Continue to move to the Optional info page. You do not need to add anything to the Optional info page, so click Save and Continue, which will take you to the Summary page. Click Back to Dashboard.
+12. Click **OAuth consent screen** again and set *Publish Status* to **Production** otherwise your credentials will expire every 7 days.
+13. Make sure **Publishing status** is set to production.
+14. Click **Credentials** in the menu on the left-hand side of the screen, then click **Create credentials** (at the top of the screen), then select *OAuth client ID*.
+15. Set the Application type to *Web application* and give this credential set a name (like "Home Assistant Credentials").
+16. Add https://my.home-assistant.io/redirect/oauth to *Authorized redirect URIs* then click **Create**. This is not a placeholder and is the URI that must be used.
+17. You will then be presented with a pop-up saying *OAuth client created* showing *Your Client ID* and *Your Client Secret*. Make a note of these (for example, copy and paste them into a text editor), as you will need these shortly. Once you have noted these strings, click **OK**. If you need to find these credentials again at any point, then navigate to *APIs & Services > Credentials*, and you will see *Home Assistant Credentials* (or whatever you named them in the previous step) under *OAuth 2.0 Client IDs*. To view both the *Client ID* and *Client secret*, click on the pencil icon; this will take you to the settings page for these credentials, and the information will be on the right-hand side of the page.
+18. Double-check that the *YouTube Data API v3* has been automatically enabled. To do this, select **Library** from the menu, then search for *YouTube Data API v3*. If it is enabled you will see *API Enabled* with a green tick next to it. If it is not enabled, then enable it.
```

Comment:
```suggestion
18. Double-check that the **YouTube Data API v3** has been automatically enabled. To do this, select **Library** from the menu, then search for **YouTube Data API v3**. If it is enabled, you will see **API Enabled** with a green tick next to it. If it is not enabled, then enable it.
```
I noticed that sometimes you use italics to markup UI strings. Any particular reason? Normally, we just use bold to markup UI strings.

Reply:
To be honest, since the way of setting up is the same as for the google-mail integration, I just copied that over and edited the APIs that are needed. When this gets merged I will update these comments on that page aswell